apps/sms/services/js/message_manager.js

```diff
@@ -41,28 +54,54 @@ var MessageManager = {
     );
   },
 
+  /**
+   * Event handler executed when a message is about to be 
+   * sent.
+   * @param {object} e The event object.
```

Comment:
__nit:__ you can take the right event description here [1] (for other event handlers as well) - eg. ```Event occurs when an SMS or MMS enters the send flow.```

Also use [1] to get the right type of the event object, so it will be like:
```@param {MozSmsEvent|MozMmsEvent} e Instance of {@link http://mdn.io/MozSmsEvent} or {@link http://mdn.io/MozMmsEvent}.```

[1] https://developer.mozilla.org/en-US/docs/Web/API/MozMobileMessageManager/onsending